src/main.zig

```diff
@@ -79,6 +84,7 @@ pub fn App(comptime Context: type, comptime config: AppConfig) type {
             const backend_type = try util.detectBackendType(allocator);
 
             try glfw.init(.{});
+            glfw.setErrorCallback(glfwErrorCallback);
```

Comment:
Should we set this before init? Since init can fail

Reply:
Done